Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Entries/FolderAssetBrowserEntry.cpp

```diff
@@ -15,27 +15,11 @@ namespace AzToolsFramework
 {
     namespace AssetBrowser
     {
-        void FolderAssetBrowserEntry::Reflect(AZ::ReflectContext* context)
-        {
-            AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
-            if (serializeContext)
-            {
-                serializeContext->Class<FolderAssetBrowserEntry, AssetBrowserEntry>()
-                    ->Field("m_isGemsFolder", &FolderAssetBrowserEntry::m_isGemsFolder)
-                    ->Version(1);
-            }
-        }
-
         AssetBrowserEntry::AssetEntryType FolderAssetBrowserEntry::GetEntryType() const
         {
             return AssetEntryType::Folder;
         }
 
-        bool FolderAssetBrowserEntry::IsGemsFolder() const
-        {
-            return m_isGemsFolder;
-        }
-
```

Comment:
note:  IsGemsFolder was never set to true by anything anymore and was not used for anything.

Previously, the folders that contained gems showed up as a Gem Icon, and showed their assets at the root level as if they were root locations.  Now, instead, it shows the actual locations in the asset browser (including drive letters, etc) so this functionality can go away.